### ISSUE TITLE
Predict time process

### ISSUE THREAD

**Issue by dgbsg** [Hongrui24/RMTPP-pytorch#4]
I have successfully run your code and got the following results.
![self-correct](https://user-images.githubusercontent.com/50312648/137244699-4a9fac7c-20fe-491c-a060-66eed2744fe3.png)

However, I find that the similarity between the predicted duration and the actual duration is very low. 
![2021-10-14 10-59-20 的屏幕截图](https://user-images.githubusercontent.com/50312648/137244714-c54b37ea-1052-493a-afb6-07594260a162.png)

Can I understand that in fact, for the time process, the model does not correctly predict the time point of an upcoming event in the future?

**Comment by waystogetthere** [comment]
That is true. I also encountered the same problem, with others' implementations or mines

**Comment by teddu7891** [comment]
You can't reproduss the result even if you have the same data set beacause we are using point process so the data change for each simulation !
